packages/datetime/examples/dateInputExample.tsx

```diff
@@ -6,17 +6,23 @@
  */
 
 import { Switch } from "@blueprintjs/core";
-import BaseExample, { handleBooleanChange, handleStringChange } from "@blueprintjs/core/examples/common/baseExample";
+import BaseExample, {
+    handleBooleanChange,
+    handleNumberChange,
+    handleStringChange,
+} from "@blueprintjs/core/examples/common/baseExample";
```

Comment:
merge conflict here. all these imports now come from `@blueprintjs/docs` (and BaseExample moves inside the `{}`, no more default).